src/librustdoc/clean/mod.rs

```diff
@@ -3121,3 +3089,30 @@ fn clean_type_binding<'tcx>(
         },
     }
 }
+
+fn clean_bound_vars<'tcx>(
+    bound_vars: &'tcx ty::List<ty::BoundVariableKind>,
+) -> Vec<GenericParamDef> {
```

Comment:
Theoretically, I could make this generic by making it take a `<T> bound_ty: ty::Binder<'tcx, T>` instead but I've decided against it for simplicity and to avoid code size explosion due to excessive monomorphization. I could use a non-generic inner function though, if you'd like.